include/openssl/blake.h

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
I thought we weren't going to expose new algorithms like this...  so this isn't the right spot for this header file.

Reply:
I was thinking that someone might want to add algorithms into their own external provider?
I will make them internal if this is not the case